### PR TITLE
Include module files as an initial commit

### PR DESCRIPTION
**Pull request by NipunaMadhushan**
## Purpose
$subject
This is the initial commit to include files related to module

### Fixes https://github.com/wso2-enterprise/internal-support-ballerina/issues/430
#2

### CODE REVIEW

File: ballerina/build.gradle [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+        }
+    }
+}
+
+task copyExternalJarsToLib(type: Copy) {
+    into "${project.projectDir}/lib"
+    from configurations.externalJars
+}
+
+task copyStdlibs(type: Copy) {
```
Comment (keizer619) [inline ballerina/build.gradle]: Shall we directly use final distribution instead of jballerina intermediate pack?

Comment (NipunaMadhushan) [reply ballerina/build.gradle]: @keizer619 addressed the suggestions.

File: ballerina/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[package]
+org = "ballerinax"
+name = "zipkin"
+version = "@toml.version@"
+distribution = "2201.7.0"
```
Comment (TharmiganK) [inline ballerina/Ballerina.toml]: Should be `2201.8.0` right?

Comment (NipunaMadhushan) [reply ballerina/Ballerina.toml]: Yes. addressed the suggestion.

**Review by TharmiganK — DISMISSED** [review]
LGTM

File: .github/workflows/daily-build.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+name: Daily build
```
Comment (keizer619) [inline .github/workflows/daily-build.yml:1]: Shall we include trivy scan for daily build as well?

Comment (NipunaMadhushan) [reply .github/workflows/daily-build.yml:1]: @keizer619 addressed the suggestions

File: native/src/main/java/io/ballerina/observe/trace/zipkin/ZipkinTracerProvider.java [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+    @Override
+    public void init() {    // Do Nothing
+    }
+
+    public static void initializeConfigurations(BString agentHostname, int agentPort, BString samplerType,
+                                                BDecimal samplerParam, int reporterFlushInterval,
+                                                int reporterBufferSize) {
+
+        String reporterEndpoint = "http://" + agentHostname + ":" + agentPort + "/api/v2/spans";
```
Comment (keizer619) [inline native/src/main/java/io/ballerina/observe/trace/zipkin/ZipkinTracerProvider.java]: @NipunaMadhushan Shall we make `http` configurable? Also can we make `https` by default?

File: native/src/main/java/io/ballerina/observe/trace/zipkin/ZipkinTracerProvider.java [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+        tracerProviderBuilder = SdkTracerProvider.builder()
+                .addSpanProcessor(BatchSpanProcessor
+                        .builder(exporter)
+                        .setMaxExportBatchSize(reporterBufferSize)
+                        .setExporterTimeout(reporterFlushInterval, TimeUnit.MILLISECONDS)
+                        .build());
+
+        tracerProviderBuilder.setSampler(selectSampler(samplerType, samplerParam));
+
+        console.println("ballerina: started publishing traces to Zipkin on " + reporterEndpoint);
```
Comment (keizer619) [inline native/src/main/java/io/ballerina/observe/trace/zipkin/ZipkinTracerProvider.java:77]: Remove System.out.println and use logger

File: native/src/main/java/io/ballerina/observe/trace/zipkin/ZipkinTracerProvider.java [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+    private static Sampler selectSampler(BString samplerType, BDecimal samplerParam) {
+        switch (samplerType.getValue()) {
+            default:
+            case "const":
+                if (samplerParam.value().intValue() == 0) {
+                    return Sampler.alwaysOff();
+                } else {
+                    return Sampler.alwaysOn();
+                }
+            case "probabilistic":
```
Comment (keizer619) [inline native/src/main/java/io/ballerina/observe/trace/zipkin/ZipkinTracerProvider.java:89]: Lets make these values constants like `RateLimitingSampler.TYPE`

File: gradle.properties [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+# Native Dependency Versions
+zipkinVersion=2.24.0
+zipkinReporterVersion=2.16.0
+okioVersion=3.5.0
+okhttpVersion=4.11.0
+kotlinVersion=1.2.71
+kotlinStdlibVersion=1.8.22
+openTelemetryVersion=1.0.0
+openTelemetryAlphaVersion=1.0.0-alpha
+guavaVersion=30.1-jre
```
Comment (keizer619) [inline gradle.properties]: Can we use later versions like 32.2-jre? 
Check for other dependencies as well

File: native/src/main/java/io/ballerina/observe/trace/zipkin/ZipkinTracerProvider.java (hunk 5 above)
Comment (NipunaMadhushan) [reply native/src/main/java/io/ballerina/observe/trace/zipkin/ZipkinTracerProvider.java:77]: addressed the suggestions

File: gradle.properties (hunk 7 above)
Comment (NipunaMadhushan) [reply gradle.properties]: updated the guava version. Other dependencies have the latest stable versions except the opentelemetry version. It has a a lot of work needs to be done to upgrade (same happened with jaeger).

File: native/src/main/java/io/ballerina/observe/trace/zipkin/ZipkinTracerProvider.java (hunk 6 above)
Comment (NipunaMadhushan) [reply native/src/main/java/io/ballerina/observe/trace/zipkin/ZipkinTracerProvider.java:89]: Addressed the suggestions

File: native/src/main/java/io/ballerina/observe/trace/zipkin/ZipkinTracerProvider.java (hunk 4 above)
Comment (NipunaMadhushan) [reply native/src/main/java/io/ballerina/observe/trace/zipkin/ZipkinTracerProvider.java]: Addressed the suggestions

**Review by keizer619 — APPROVED** [review]

**Pull request merged**